### ISSUE TITLE
Accessibility for blind players

### ISSUE THREAD

**Issue by Nuktos** [Lattyware/massivedecks#558]
## Describe the feature
A clear and concise description of what the feature you would like is.

We're currently trying to get a new friend to play with us. Our main problems right now are that the game regularly refresh itself, cancelling the selection of cards before confirmation, that the confirmation button is apparently hard to find, which makes it necessary to have more time to confirm (so it goes back to the first problem), and that the black cards's text is cut word by word, so the blind friend has to make the screen reader read each word manually. We wanted to make a game for december 31st like every year, but oh, well, I hope you'll be able to do it in a couple months at least?

Thank you anyway, your game is already doing a lot!
